clients/drcachesim/tracer/tracer.cpp

```diff
@@ -940,16 +948,23 @@ insert_filter_addr(void *drcontext, instrlist_t *ilist, instr_t *where, user_dat
             XINST_CREATE_load_int(drcontext, opnd_create_reg(reg_ptr),
                                   OPND_CREATE_INT32(mask)));
 #endif
+#ifdef RISCV64
+    ASSERT(false, "NYI on RISCV64");
```

Comment:
I assume we're adding all these asserts because the platform independent macros used here are actually not yet implemented? So this PR doesn't implement all such macros?